test-packages/integration-tests/test/cache.spec.ts

```diff
@@ -35,11 +35,11 @@ describe('CacheDestination & CacheClientCredentialToken', () => {
       zid: 'provider_token',
```

Comment:
[q] can we not use the provider service/user tokens we have in the `mocked-access-tokens.ts`?

Reply:
good idea, I'll check

Reply:
Or not, because I now deleted the file as discussed.